neqo-transport/src/recovery.rs

```diff
@@ -324,12 +524,20 @@ impl LossRecovery {
 
         self.pto_count = 0;
 
-        let acked_packets = acked_packets
-            .into_iter()
-            .map(|(_k, v)| v)
-            .collect::<Vec<_>>();
+        self.cc
+            .on_packets_acked(acked_packets.iter().map(|(_k, v)| v));
+
+        self.cc.on_packets_lost(
+            now,
+            prev_largest_acked_sent_time,
+            self.rtt_vals.pto(),
+            &lost_packets,
+        );
 
-        (acked_packets, lost_packets)
+        (
+            acked_packets.into_iter().map(|(_, p)| p).collect(),
```

Comment:
```suggestion
            acked_packets.values().collect(),
```

Reply:
This makes me wonder whether `remove_acked` should just return `Vec` rather than the `BTreeMap`.

Reply:
Ok let's try it. My only worry is that we're now being a little less connected between the largest acked pkt PN and entries in the acked_packets vec. Before we'd do a `get` and now we implicitly assume it's the last entry in the vec. Looks like a safe assumption, though.